teamwork/templates/base.html

```diff
@@ -91,6 +93,24 @@
       {% endblock %}
     </main>
 
+    <!-- Add project popup -->
+    <div class="modal fade bs-example-modal-sm" tabindex="-1" role="dialog">
+                <div class="modal-dialog" role="document">
+                 <div class="modal-content">
+                  <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                    <h4 class="modal-title">Add new project</h4>
+                  </div>
+                  <form action="add_project" method="post" class="modal-body">
+                    <input id="new_project_name" name="new_project_name" type="text" class="form-control" placeholder="New project name">
+                  </form>
+                  <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                    <button type="button" class="btn btn-primary">Add project</button>

```

Comment:
Jotta `<button type="button" class="btn btn-primary">Add project</button>` lähettäisi mitään tietoja kun sitä painaa, niin sen on oltava `form`in sisällä ja siinä on oltava `type="submit"` attribuutti.